tests/rustdoc-ui/2024-doctests-checks.stdout

```diff
@@ -1,12 +1,7 @@
 
-running 1 test
-test $DIR/2024-doctests-checks.rs - Foo (line 8) ... ok
-
-test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in $TIME
-
-
-running 1 test
+running 2 standalone doctests
```

Comment:
Seems like there is a regression somewhere here. ^^'

Reply:
yeah i broke merged doctests, and instead of getting an error anywhere, everything just silently reverted to standalone doctests.  feels like maybe there should be a perma-unstable option to force merged doctests so we can use it in tests like this.

especially since we're doing string-based codegen, which doesn't play the best with IDEs.